### PR TITLE
GMX-1598 collapse mobile ads

### PR DESCRIPTION
**Pull request by mtyeh411**
EXAMPLE

before:
screen shot 2016-11-30 at 4 53 23 pm

after:
screen shot 2016-11-30 at 4 53 13 pm

### CODE REVIEW

**Comment by mtyeh411** [comment]
@weddingwire/gmx @mjn298 please review

**Comment by americos** [comment]
looks good

File: lib/dfp_helper.rb [hunk 1]
```diff
@@ -105,7 +105,9 @@ def dfp_helper_mobile_tag(_i, options = {})
 <div id='#{_id}' class='mobile-ad #{options[:div_class]}'>
 <script type='text/javascript'>
 googletag.cmd.push(function() {
-googletag.defineSlot('#{_i}', #{options[:size]}, '#{options[:div_id]}').addService(googletag.pubads());
+googletag.defineSlot('#{_i}', #{options[:size]}, '#{options[:div_id]}')
+.setCollapseEmptyDiv(true)
```
Comment (nankrah) [inline lib/dfp_helper.rb]: how does this affect desktop?

Comment (mtyeh411) [inline lib/dfp_helper.rb]: desktop has its own helper which does this above

**Comment by mtyeh411** [comment]
@chancedowns can you merge? or grant access to @nankrah and i for this repo?

**Comment by mtyeh411** [comment]
actually @chancedowns can you not merge?  i'm seeing an instance of an ad that shouldn't collapse automatically (the ad changes so you see a flash of content at that time), so i'll need to ad an option to the helper to not collapse

but if you could grant merge right that'd be helpful :)

**Comment by mtyeh411** [comment]
@chancedowns @weddingwire/gmx updated this to only collapse when option is passed in

**Comment by mtyeh411** [comment]
blocks https://github.com/weddingwire/mobile-ng/pull/1550

**Pull request merged**